tests/integration/update.yaml

```diff
@@ -37,10 +37,6 @@ policy:
       trust_root: sse
       delegations: ["belitzphilipp", "starkteetje"]
   - pattern: "docker.io/securesystemsengineering/connaisseur:*"
-    validator: dockerhub
-    with:
-      trust_root: sse
-  - pattern: "docker.io/securesystemsengineering/connaisseur:helm-hook-*"
     validator: allow
```

Comment:
I see why this is necessary here. But I would strongly recommend that we run a test that the update to the new connaisseur image works upon release in the release workflow.
Can be minimal just like:
- grep last and new release
- install last with last preconfig
- get new charts
- upgrade